src/metadata/routes/comments.clj

```diff
@@ -5,6 +5,37 @@
         [ring.util.http-response :only [ok]])
   (:require [metadata.services.comments :as comments]))
 
+(defroutes admin-comment-routes
+  (context "/admin/comments" []
+    :tags ["admin-comments"]
+
+    (GET "/:commenter-id" []
+      :path-params [commenter-id :- CommenterId]
+      :query [{:keys [user]} StandardUserQueryParams]
+      :responses {200      {:schema      CommentDetailsList
+                            :description "Comment details are listed in the response"}
+                  500      {:schema      ErrorResponseUnchecked
+                            :description "Unchecked errors"}
+                  :default {:schema      ErrorResponse
+                            :description "All other errors"}}
+      :summary "List All Comments Added by a User"
+      :description "This endpoint allows an administrator to list all comments that were entered by a single user."
+      (ok (comments/list-user-comments commenter-id)))
+
+    (DELETE "/:commenter-id" []
+      :path-params [commenter-id :- CommenterId]
+      :query [{:keys [user]} StandardUserQueryParams]
+      :responses {200      {:schema      (describe String "Empty response")
```

Comment:
This is kind of hokey, but it was the only way that I found to get compojure-api not to complain about the empty response when using this type of endpoint definition.

Reply:
The `DELETE /user/:tag-id` endpoint is using something like this:
```clojure
200 {:schema      nil
     :description "The tag was successfully deleted"}
```
Would that work with the `DELETE` endpoints you added? Except the responses would probably have to be just `(ok)`. 👌

Reply:
Ah! I finally spotted the difference!

```clojure
       :coercion middleware/no-response-coercion
```

Adding this allowed me to return a truly empty response.

Reply:
Ah, yeah, I forgot about that requirement too 😖